### PR TITLE
UG: explain product update editClient

### PR DESCRIPTION
**Pull request by Thing1Thing2**
image

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -399,6 +399,7 @@ Format: `editClient i/INDEX [n/NAME] [p/PHONE_NUMBER] [a/ADDRESS] [e/EMAIL] [b/B
 * `INDEX` **must be a positive integer** 1, 2, 3, …​
 * At least one optional detail must be modified.
 * Maintain value of details not edited by the command.
+* Editing the client's products removes all existing products and adds all products specified.
```
Comment (sikai00) [inline docs/UserGuide.md:402]: perhaps a more succinct way to put it is 'replaces', but this is clear enough.

**Review by sikai00 — APPROVED** [review]

**Review by ThomasHoooo — COMMENTED** [review]
I think it doesn't mention the use that it can take in multiple product prefixes?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -323,7 +323,7 @@ Format: `addClient n/NAME p/PHONE_NUMBER [a/ADDRESS] [e/EMAIL] [b/BIRTHDAY] [pd/
 * `PHONE_NUMBER` should contain only numbers and be at least 8 digits long.
 * `EMAIL`, `BIRTHDAY`, `ADDRESS` and `PRODUCT` are optional.
 * `BIRTHDAY` in the future are not acceptable.
-* `PRODUCT` must exists already.
+* `PRODUCT` must exists already. Reuse `pd\` prefix to add each product to the client.
```
Comment (ThomasHoooo) [inline docs/UserGuide.md:326]: I think this description is not clear enough. A better way could be `You can add multiple products to the client`.
Then add one example to show how the user can add multiple products. Please do the same for `editClient` as well.

Comment (Thing1Thing2) [reply docs/UserGuide.md]: Ok, made the changes. Please have a look, thank you.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -323,7 +323,7 @@ Format: `addClient n/NAME p/PHONE_NUMBER [a/ADDRESS] [e/EMAIL] [b/BIRTHDAY] [pd/
 * `PHONE_NUMBER` should contain only numbers and be at least 8 digits long.
 * `EMAIL`, `BIRTHDAY`, `ADDRESS` and `PRODUCT` are optional.
 * `BIRTHDAY` in the future are not acceptable.
-* `PRODUCT` must exists already.
+* `PRODUCT` must exists already. You can add multiple products to the client. Reuse `pd\` prefix to add each product to the client.
```
Comment (ThomasHoooo) [inline docs/UserGuide.md:326]: it should be `pd/`

File: docs/UserGuide.md [hunk 4]
```diff
@@ -426,6 +428,7 @@ Format: `editClient i/INDEX [n/NAME] [p/PHONE_NUMBER] [a/ADDRESS] [e/EMAIL] [b/B
 * At least one optional detail must be modified.
 * Maintain value of details not edited by the command.
 * If you wish to edit a client's `NAME` and the `NAME` already exists in _MyInsuRec_, it will result in an error!
+* Editing the client's products replaces all existing products with the products specified. You can add multiple products to the client. Reuse `pd\` prefix to add each product to the edited client.
```
Comment (ThomasHoooo) [inline docs/UserGuide.md:431]: same issue

File: docs/UserGuide.md (hunk 3 above)
Comment (Thing1Thing2) [reply docs/UserGuide.md:326]: ok

**Review by ThomasHoooo — APPROVED** [review]
LGTM

**Pull request merged**